src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngineResolver.cs

```diff
@@ -36,13 +45,9 @@ public override bool Lookup(DName name, out NameLookupInfo nameInfo, NameLookupP
             {
                 var data = fi;
                 var type = fi._type._type;
+                var description = $"{str} variable";
 
-                nameInfo = new NameLookupInfo(
-                    BindKind.PowerFxResolvedObject,
-                    type,
-                    DPath.Root,
-                    0,
-                    data);
+                nameInfo = new NameLookupInfo(BindKind.PowerFxResolvedObject, type, DPath.Root, 0, data, displayName: new DName(description));                
```

Comment:
Can you put this in a helper so that it's shared with above? Lot of flags here that are easy to get different. 

NameLookupInfo Create(string name, RecalcFormulaInfo fo) {
   var data = fi;
                var type = fi._type._type;
                var description = $"{str} variable";

                nameInfo = new NameLookupInfo(BindKind.PowerFxResolvedObject, type, DPath.Root, 0, data, displayName: new DName(description));                
             
   return nameInfo;
}